samples/docview/Info.plist.in

```diff
@@ -61,27 +50,27 @@
 	</array>
 	<key>CFBundleExecutable</key>
 	<string>EXECUTABLE</string>
+	<key>CFBundleGetInfoString</key>
+	<string>EXECUTABLE version VERSION, (c) 2002-2021 wxWidgets</string>
 	<key>CFBundleIconFile</key>
 	<string>wxmac.icns</string>
+	<key>CFBundleIdentifier</key>
+	<string>org.wxwindows.IDENTIFIER</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleLongVersionString</key>
+	<string>VERSION, (c) 2002-2021 wxWidgets</string>
 	<key>CFBundleName</key>
 	<string>EXECUTABLE</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
-	<key>CFBundleSignature</key>
-	<string>WXMA</string>
-	<key>CFBundleVersion</key>
-	<string>VERSION</string>
 	<key>CFBundleShortVersionString</key>
 	<string>VERSION</string>
-	<key>CFBundleGetInfoString</key>
-	<string>EXECUTABLE version VERSION, (c) 2002-2008 wxWidgets</string>
-	<key>CFBundleLongVersionString</key>
-	<string>VERSION, (c) 2002-2008 wxWidgets</string>
+	<key>CFBundleVersion</key>
+	<string>VERSION</string>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright 2002-2008 wxWidgets</string>
-	<key>LSRequiresCarbon</key>
-	<true/>
-	<key>CSResourcesFileMapped</key>
-	<true/>
+	<string>Copyright 2002-2021 wxWidgets</string>
+	<key>NSPrincipalClass</key>
+	<string>wxNSApplication</string>
```

Comment:
shouldn't we use the CMake Vars here as well ?

Reply:
Oops, we totally should, I forgot to update this file, thanks for noticing!